lua/engine.lua

```diff
@@ -84,10 +84,8 @@ Engine.load = function(name, callback)
   if type(callback) == 'function' then
     norns.report.did_engine_load = function()
       --print("Engine: norns.report.did_engine_load callback")
-      local status = norns.try(callback,"init")
-      norns.init_done(status)
+      callback()
```

Comment:
curious why not `norns.try` here?

Reply:
because the init function (`Script.init`) called higher up has its own `try`